specs/creditCard.spec.js

```diff
@@ -27,6 +27,10 @@ describe('CreditCard', () => {
     it('should return true with its a INVALID month', () => {
       expect(creditcard.isExpirationDateValid('15', '2020')).toBeFalsy();
     });
+
+    it('should return false with empty month or year', () => {
```

Comment:
Create scenario to year and month. e.g. `Should not be expiration date valid when month is invalid`, `Should not be expiration date valid when year is invalid`